diagnostic.rb

```diff
@@ -2,15 +2,26 @@
 # Diagnostic answers to:
 
 # question 2:
-
+Licensee.create surname: 'Bishop',
+                        given_name: 'Caren',
+                        gender: 'f',
+                        height: 63,
+                        weight: 132 ,
+                        born_on: '1943-09-26',
+                        eye_color: 'Brown',
+                        hair_color: 'Black'
 # question 3:
-Licensee.transaction do
-  CSV.foreach 'data/licensees.csv', headers: true do |row|
+Licensee.transaction dosss
+  CSV.foreach(Rails.root + 'data/licensees.csv',
+      headers: true) do |row|
+        licensee = row.to_hash
+                      Licensee.create! licensee unless Licensee.exists? licensee
   end
 end
 
 # question 4:
-
-# question 5:
+Licensee.where eye_color: 'Hazel', hair_color: 'Brown', hair_color: 'Black'
```

Comment:
In this case, you could do the following:
`Licensee.where eye_color: 'Hazel', hair_color: ['Brown', 'Black']`
But if you wanted to do an OR on a combination of attributes, the best option is (currently) to use a query.